easybuild/easyconfigs/p/Python/Python-2.7.11-CrayGNU-2015.11-XC.eb

```diff
@@ -0,0 +1,75 @@
+# contributed by Luca MArsella and GPPezzi (CSCS)
+
+name = 'Python'
+version = "2.7.11"
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+
+toolchain = {'name': 'CrayGNU', 'version': '2015.11-XC'}
+toolchainopts = {'dynamic': True, 'pic': True, 'opt': True, 'optarch': True}
+
+numpyversion = '1.10.4'
+scipyversion = '0.16.1'
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+patches = ['unixccompiler-Python-2.7.11.patch']
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
+    ('ncurses', '5.9'),
+    ('freetype', '2.5.5'),
+    ('libpng', '1.6.16'),
+]
+
+osdependencies = [('libopenssl-devel')]
+
+buildopts = 'LINKCC="$CC -dynamic"'
+
+# order is important!
+# package versions updated Mar 1st 2016
+exts_list = [
+    ('setuptools', '20.2.2', {
+        'source_urls': ['http://pypi.python.org/packages/source/s/setuptools/'],
+    }),
+    ('pip', '8.0.3', {
+        'source_urls': ['http://pypi.python.org/packages/source/p/pip/'],
+    }),
+    ('nose', '1.3.7', {
+        'source_urls': ['http://pypi.python.org/packages/source/n/nose/'],
+    }),
+    ('numpy', numpyversion, {
+        'source_urls': [('http://sourceforge.net/projects/numpy/files/NumPy/%s' % numpyversion, 'download')],
+        'patches': [
+            'numpy-1.8.0-mkl.patch', # % numpyversion,
+        ],
+    }),
+    ('scipy', scipyversion, {
+        'source_urls': [('http://sourceforge.net/projects/scipy/files/scipy/%s' % scipyversion, 'download')],
+    }),
+    ('mpi4py', '1.3.1', {
+        'source_urls': ['http://bitbucket.org/mpi4py/mpi4py/downloads/'],
+    }),
+    ('Cython', '0.23.4', {
+        'source_urls': ['http://www.cython.org/release/'],
+    }),
+    ('six', '1.10.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/six/'],
+    }),
+    ('virtualenv', '14.0.5', {
+        'source_urls': ['https://pypi.python.org/packages/source/v/virtualenv'],
+    }),
+    ('matplotlib', '1.5.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/m/matplotlib/'],
+    }),
+    ('pandas', '0.17.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pandas/'],
+    }),
```

Comment:
current Python easyconfigs include a couple of extra extensions: `blist`, `pbr`, `dateutil`, `mock`, `ecdsa`, `pytz`, ...